Updates regarding RHDEVDOCS-4758 (#55298)
openshift/openshift-docs: modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

.Prerequisites

* An OpenShift web console session with cluster administrator privileges.
* The Developer perspective is enabled.

.Procedure

. In the *Administrator* perspective, navigate to *Administration* -> *Cluster Settings*.
. Select the *Configuration* tab and click the *Console (operator.openshift.io)* resource.
. Click *Actions* -> *Customize*.

> . In the *Pre-pinned navigation items*, *Add page*, and *Developer Catalog* sections, enable or disable the items in the list.

Reviewer: Maybe 
```suggestion
. Enable or disable items in the *Pre-pinned navigation items*, *Add page*, and *Developer Catalog* sections.
```

[NOTE]
====

> As an administrator, you can define which navigation items will display by default for all users. You can also reorder the navigation items.

Reviewer: ```suggestion
As an administrator, you can define the navigation items that appear by default for all users. You can also reorder the navigation items.
```
* IBM Style: "Write in the simple present tense as much as possible."
* IBM Style: "display verb: Use only as a transitive verb. For example, write "the system displays a message" or "the message is displayed" (not "the message displays")."
